apps/web/app/utils/post-utils.ts

```diff
@@ -0,0 +1,28 @@
+import { removeTrailingPeriod } from './string-utils';
+
+/**
+ * Get a formatted post prompt without trailing periods
+ * @param prompt The prompt content from a post
+ * @returns The formatted prompt
+ */
+export function formatPostPrompt(prompt: string | undefined | null): string {
```

Comment:
These functions all do exactly the same thing, `removeTrailingPeriod` - what's the point of them?

Reply:
I'll consolidate in the next commit